posydon/binary_evol/singlestar.py

```diff
@@ -314,14 +314,17 @@ def to_oneline_df(self, history=True, prefix='', **kwargs):
         for name in scalar_names:
             if hasattr(self, name):
                 if name == 'natal_kick_array':
-                    natal_kick_array = getattr(self, name)
+                    natal_kick_array = np.array(getattr(self, name), dtype=np.float64)
                     for i in range(4):
                         col_name = prefix+name+'_{}'.format(int(i))
                         oneline_df[col_name] = [natal_kick_array[i]]
+                elif (name == 'spin_orbit_tilt') | (name == 'f_fb'):
+                    oneline_df[prefix+name] = [np.float64(getattr(self, name))]
```

Comment:
Did you checked, that there are only 'spin_orbit_tilt' and 'f_fb', which need to get float64? I'd expect there should be more floats.